io/disk/ltp_fs.py.data/ltp_fs.yaml

```diff
@@ -0,0 +1,6 @@
+general:
+    disk: '/dev/sda'
+    script: 'runltp'
+    args: '-s fs_di,-s fs_inod'
+    mount_point: '/home'
```

Comment:
would be better if mount point can be dynamically created by the script, and not as an input from the user.